src/Altinn.Platform/Altinn.Platform.Events/Tests/Mocks/QueueServiceMock.cs

```diff
@@ -10,6 +11,30 @@ namespace Altinn.Platform.Events.Tests.Mocks
 {
     public class QueueServiceMock : IQueueService
     {
+        public QueueServiceMock()
+        {
+            OutboundQueue = new Dictionary<string, List<CloudEventEnvelope>>();
+        }
+
+        /// <summary>
+        /// Quee
```

Comment:
> Quee

We have a GlobalSuppressions.cs file that should make it okay to have public methods and properties without comments in the Test project(s).